spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java

```diff
@@ -516,6 +590,48 @@ private static void deleteManifests(FileIO io, List<ManifestFile> manifests) {
         .run(item -> io.deleteFile(item.path()));
   }
 
+  // Attempt to use Spark3 Catalog resolution if available on the path
```

Comment:
This is all just moved verbatim out of BaseSparkAction right?

Reply:
Yes